### PR TITLE
GEOMESA-1045 Adding hbase geoserver install

### PR DESCRIPTION
**Pull request by elahrvivaz**
Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

**Comment by elahrvivaz** [comment]
I think we need some more docs on hbase - geoserver starts up with this change, but I am not sure how to test showing actual points. @jnh5y any suggestions?

**Comment by jahhulbert-ccri** [comment]
i guess i should try this technically before merging :) i'll see if i can get to it soon.

**Comment by jahhulbert-ccri** [comment]
@elahrvivaz do we have a geomesa hbase tutorial or anything like that?

**Comment by elahrvivaz** [comment]
I couldn't find an hbase tutorial... I put what worked for me in the README, but having someone else test it out would be good. I am going to go back today and test it again - I didn't actually get points showing up, but I was able to add a data store and empty layer.

File: geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseFeatureReader.scala [hunk 1]
```diff
@@ -9,46 +9,45 @@
...
+  }
 
-  val scanners =
-    scans
-      .map { s => table.getScanner(s) }
+  val scanners = scans.map(table.getScanner)
 
   val results =
-    Try {
+    try {
```
Comment (jahhulbert-ccri) [inline geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseFeatureReader.scala]: YES

**Comment by jahhulbert-ccri** [comment]
428dbf19bc437c9d2f594e5db2175e042c23d59f

**Pull request closed without merging**